django_socio_grpc/protobuf/proto_classes.py

```diff
@@ -87,6 +88,8 @@ class ProtoField:
     def field_type_str(self) -> str:
         if isinstance(self.field_type, str):
             return self.field_type
+        if isinstance(self.field_type, type) and issubclass(self.field_type, Enum):
```

Comment:
Add a comment why we need a sub Enum

Reply:
I don't understand what info you want here. It's just that to check if we have an Enum i use issubclass and not isinstance because an enumeration is never an instance

Reply:
I'm talking about why we use a message scoped Enum in .proto (Message.Enum) instead of just Enum